cycler.py

```diff
@@ -335,7 +335,11 @@ def __repr__(self):
     def _repr_html_(self):
         # an table showing the value of each key through a full cycle
         output = "<table>"
-        sorted_keys = sorted(self.keys)
+        try:
+            sorted_keys = sorted(self.keys)
```

Comment:
my only concern is that are we guaranteed order between successive iterations through an unchanged set() object? So, if one makes a list of reprs from a set object, and then does `zip(set_object, list_object)`, will the items pair up correctly every time?

I might be a little overly paranoid about things like that, though...